SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -84,3 +93,34 @@ void FSpatialGDKEditor::GenerateSnapshot(UWorld* World, FString SnapshotFilename
 		FailureCallback.ExecuteIfBound();
 	}
 }
+
+void FSpatialGDKEditor::LoadAllStreamingLevels(UWorld* World)
+{
+	const TArray<ULevelStreaming*> StreamingLevels = World->GetStreamingLevels();
```

Comment:
Would we ever want to unload these levels once the schema generation is complete?